lib/args.js

```diff
@@ -1,5 +1,8 @@
 "use strict";
 
+// Permanent Comment: Hack to enable `-h` output in dev to use `multibot` name.
+// process.argv[1] = "multibot";
```

Comment:
When I want to regenerate the usage message for the readme, I run `bin/multibot.js -h` from my branch. Unfortunately, instead of `multibot` as the script name it becomes `bin/multibot.js`, which then affects the padding for things like the examples, so I can't then search and replace before pasting in the readme.

This allows me to hack in the script name I want that reflects what it will be after a real npm install.